studybuddies/src/app/app.module.ts

```diff
@@ -1,16 +1,41 @@
+import { RouterModule, Routes } from '@angular/router';
+
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
-import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { StudentComponent } from './student/student.component';
+import { TutorComponent } from './tutor/tutor.component';
+import { ClassListStudentComponent } from './student/class-list-student/class-list-student.component';
+import { CreateClassStudentComponent } from './student/create-class-student/create-class-student.component';
+import { ClassListTutorComponent } from './tutor/class-list-tutor/class-list-tutor.component';
+import { CreateClassTutorComponent } from './tutor/create-class-tutor/create-class-tutor.component';
+import { HomeComponent } from './home/home.component';
+
+const routes: Routes = [
+  { path: 'student', component: StudentComponent },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155411)
```suggestion
  { path: "student", component: StudentComponent },
```